src/main.rs

```diff
@@ -61,6 +64,16 @@ impl Worker {
     }
 }
 
+macro_rules! first_column {
```

Comment:
Macros seem really hacky, but my goal was to just declare the format strings that `println!()` uses in one place.